console/my-account/how-to/use-2fa.mdx

```diff
@@ -36,19 +36,37 @@ Download the app of your choice and install it onto your smartphone.
     <Lightbox src="scaleway-org-dropdown-myprofile.webp" alt="" />
 2. Scroll to the bottom of your profile page, and click **Enable MFA**. A pop-up displays.
     <Lightbox src="scaleway-2FA-popup-1.webp" alt="" />
-3. Enter the code shown on the popup into your MFA app, or alernatively, scan the QR code into your app. 
-    
+3. Enter the code shown on the popup into your MFA app, or alernatively, scan the QR code into your app.
     Your app sets up MFA for your Scaleway account, and displays a 6-digit code.
 4. Enter the 6-digit code given by your app into the pop-up box, and click **Submit**. A pop-up displays:
     <Lightbox src="scaleway-2FA-popup-2.webp" alt="" />
-5. Download or copy the backup codes shown, and save them in a safe place. These codes will be the only way you can regain access to your account without your MFA app. 
+5. Download or copy the backup codes shown, and save them in a safe place. These codes will be the only way you can regain access to your account without your MFA app.
 
 MFA is now enabled. Next time you log in to your Scaleway account, you will be prompted for a MFA code. Use your MFA app to get the code and enter it into the Scaleway console.
 
+## How to update MFA
+
+If you no longer have access to the device in which you set up your MFA, you can update your MFA information and set up via the Scaleway console.
+
+<Message type="requirement">
+  - You have [enabled MFA](#how-to-enable-mfa)
+</Message>
+
+1. Click to open the Organization drop-down menu in the top-right corner of the console, and select **My Profile**.
+2. Scroll to the bottom of your profile page, and click **Update MFA**. A pop-up displays.
+3. Enter the code shown on the pop-up into your MFA app, or alernatively, scan the QR code into your app.
+    Your app sets up MFA for your Scaleway account, and displays a 6-digit code.
+4. Enter the 6-digit code given by your app into the pop-up box, and click **Submit**. A pop-up displays:
+5. Download or copy the backup codes shown, and save them in a safe place. These codes will be the only way you can regain access to your account without your MFA app.
+
 ## How to disable MFA
 
+<Message type="important">
+  You cannot disable MFA because you are a member of one or more Organizations where MFA is enforced. If you wish to disable MFA, you must first leave these Organizations. If you do not know which of your Organizations enforce MFA, follow the procedure below until step 2. The Organizations will be listed in the **Disable MFA** pop-up.
```

Comment:
Should this message say something more like "You cannot disable MFA **if** you are a member of" bla bla?